boards/st/stm32u5g9j_dk2/stm32u5g9j_dk2.dts

```diff
@@ -311,6 +327,42 @@ zephyr_udc0: &usbotg_hs {
 	};
 };
 
+&xspi1 {
+	clocks = <&rcc STM32_CLOCK(AHB2_2, 12U)>,
+		<&rcc STM32_SRC_PLL2_Q HSPI_SEL(2)>;
```

Comment:
Nitpick
```suggestion
		 <&rcc STM32_SRC_PLL2_Q HSPI_SEL(2)>;
```

Reply:
Try to configure your editor for 8-char wide tabulation when you're editing Zephyr C source files, header files or DTS files.